### PR TITLE
Implements scrolling columns on the task view

### PR DESCRIPTION
**Pull request by rauhryan**
Merges into #327

### CODE REVIEW

File: app/assets/stylesheets/_milestone.scss [hunk 1]
```diff
@@ -16,7 +16,6 @@
     position: relative;
 
     .ui-icon {
-      float:left;
```
Comment (dahlbyk) [inline app/assets/stylesheets/_milestone.scss]: I'm curious what motivates this change... seems unrelated?

Comment (rauhryan) [inline app/assets/stylesheets/_milestone.scss]: tinkering I noticed it didn't need to be floated.

For some reason the float was messing with the header heights somehow

File: ember-app/app/templates/components/columns/hb-milestone.hbs [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <div class="column-contents">
+
```
Comment (discorick) [inline ember-app/app/templates/components/columns/hb-milestone.hbs]: Just a nitpick :D - empty commit on this file

**Comment by rauhryan** [comment]
@discorick done

**Comment by dahlbyk** [comment]
I'm seeing an extra scroll bar in Firefox that's not present on Milestones:
firefox_2016-08-03_13-41-56
firefox_2016-08-03_13-42-01

**Comment by rauhryan** [comment]
Cool, I'll take a look

**Comment by rauhryan** [comment]
@huboard/committers :boom:

Let's merge this baby
